kDrive/UI/Controller/Menu/SwitchUserViewController.swift

```diff
@@ -141,17 +141,35 @@ extension SwitchUserViewController: UITableViewDataSource {
 
     func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
         let account = accountManager.accounts[indexPath.row]
-        let cell = tableView.dequeueReusableCell(type: UserAccountTableViewCell.self, for: indexPath)
-        cell.initWithPositionAndShadow(isFirst: true, isLast: true)
-        cell.titleLabel.text = account.user.displayName
-        cell.userEmailLabel.text = account.user.email
-        cell.logoImage.image = KDriveResourcesAsset.placeholderAvatar.image
-        cell.isUserInteractionEnabled = !driveInfosManager.getDrives(for: account.userId).isEmpty
-
-        account.user.getAvatar { image in
-            cell.logoImage.image = image
+        if account == accountManager.currentAccount {
+            let cell = tableView.dequeueReusableCell(type: UserAccountTableViewCell.self, for: indexPath)
+            cell.accessoryImageView.image = KDriveResourcesAsset.check.image
+            cell.initWithPositionAndShadow(isFirst: true, isLast: true)
+            cell.titleLabel.text = account.user.displayName
+            cell.userEmailLabel.text = account.user.email
+            cell.logoImage.image = KDriveResourcesAsset.placeholderAvatar.image
+            cell.isUserInteractionEnabled = !driveInfosManager.getDrives(for: account.userId).isEmpty
+            cell.isUserInteractionEnabled = false
+
+            account.user.getAvatar { image in
+                cell.logoImage.image = image
+            }
+
+            return cell
+        } else {
+            let cell = tableView.dequeueReusableCell(type: UserAccountTableViewCell.self, for: indexPath)
+            cell.initWithPositionAndShadow(isFirst: true, isLast: true)
+            cell.titleLabel.text = account.user.displayName
+            cell.userEmailLabel.text = account.user.email
+            cell.logoImage.image = KDriveResourcesAsset.placeholderAvatar.image
+            cell.isUserInteractionEnabled = !driveInfosManager.getDrives(for: account.userId).isEmpty
+
+            account.user.getAvatar { image in
+                cell.logoImage.image = image
+            }
+
+            return cell
```

Comment:
The whole block is duplicated 

Simply use ternary conditions for 
`cell.isUserInteractionEnabled` and `cell.accessoryImageView.image`